src/cdk/v2/destinations/bloomreach_catalog/rtWorkflow.yaml

```diff
@@ -24,7 +24,7 @@ steps:
 
   - name: processRecordEvents
     template: |
-      $.processRecordInputs(^.{.message.type === $.EventType.RECORD}[], ^[0].destination)
+      await $.processRecordInputs(^.{.message.type === $.EventType.RECORD}[], ^[0].destination)
```

Comment:
@koladilip is this valid? are we testing cdkv2 destinations?

Reply:
we can use await in cdk v2